etl/planning_data/README.MD

```diff
@@ -0,0 +1,33 @@
+Following scripts should be scheduled to run regularly to load livestream data into database.
+
+```
+# querying API to obtain data
+python3 obtain_livestream_data.py > all_data.json
+
+# loading data into Colouring database
+python3 load_into_database
+
+# removing tile cache for planning_applications_status layer - note that location of cache depends on your configuration
+rm /srv/colouring-london/tilecache/planning_applications_status/* -rf
+```
+
+As loading into databases expects environment variables to be set, one option to actually schedult it in a cron is something like
```

Comment:
```suggestion
As loading into databases expects environment variables to be set, one option to actually schedule it in a cron is something like
```

Reply:
Is this something you could update in https://github.com/colouring-cities/colouring-london-config such that it would be one of the scripts run during deployment?